### PR TITLE
[WC-2670]: Fix popup exclusion variables

### PR DESCRIPTION
**Pull request by rahmanunver**
### Pull request type

Refactoring (e.g. file rename, variable rename, etc.)

Bug fix (non-breaking change which fixes an issue)

---

### Description
Moved widget styling to atlas to make use of exclusion variables.
https://github.com/mendix/atlas/pull/163/files

<!--
### What should be covered while testing?
Toggling $exclude-pop-up-menu should allow to have an unstyled widget.

### CODE REVIEW

File: packages/pluggableWidgets/popup-menu-web/src/ui/PopupMenuPreview.scss
Comment (r0b1n) [inline packages/pluggableWidgets/popup-menu-web/src/ui/PopupMenuPreview.scss:1]: Atlas styles are present in the preview as well. So in theory only the styles that are fixing preview have to be here, the rest is automatically coming from atlas, so should be no need to duplicate them.

**Review by r0b1n — DISMISSED** [review]

**Review by r0b1n — APPROVED** [review]

File: packages/pluggableWidgets/popup-menu-web/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
```
Comment (leonardomendix) [inline packages/pluggableWidgets/popup-menu-web/CHANGELOG.md:9]: I'd say we can change that to a breaking change.

Comment (rahmanunver) [reply packages/pluggableWidgets/popup-menu-web/CHANGELOG.md]: changed 👍

File: packages/pluggableWidgets/popup-menu-web/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Breaking changes
+
+-   We moved the widget styling to Atlas core to make exclusion variables work. Please update atlas-core module to keep the styling intact.
```
Comment (gjulivan) [inline packages/pluggableWidgets/popup-menu-web/CHANGELOG.md:11]: let's specify minimum version here.
atlas-core 3.15.0 for Studio Pro 10.6.0 and above or 3.12.5 for below.

**Review by leonardomendix — APPROVED** [review]

**Review by leonardomendix — APPROVED** [review]

**Review by gjulivan — APPROVED** [review]

**Review by gjulivan — APPROVED** [review]

**Pull request merged**